core/src/main/resources/hudson/model/CauseAction/summary.jelly

```diff
@@ -25,13 +25,31 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <t:summary icon="orange-square.png">
-    <j:forEach var="entry" items="${it.causeCounts.entrySet()}">
-      <p>
-        <st:include page="description.jelly" it="${entry.key}"/>
-        <j:if test="${entry.value > 1}">
-          <st:nbsp/>${%Ntimes(entry.value)}
-        </j:if>
-      </p>
-    </j:forEach>
+  <h4>Build was started by</h4>   
+	<style type="text/css">
+	div.tab:not(:target) div.content, div.tab:target div.show, div.tab div.hide {
+		display: none;
+	}
+	div.tab:target div.content,  div.tab:target div.hide {
+		display: block;
+	}
+	</style>
+	<div class="tab" id="text1">
+		<div class="show">		
+			<a href="#text1">show details</a>
```

Comment:
The strings should be localizable: https://wiki.jenkins-ci.org/display/JENKINS/Internationalization